app/statistics/server/functions/get.js

```diff
@@ -148,3 +148,15 @@ statistics.get = function _getStatistics() {
 
 	return statistics;
 };
+
+statistics.getConnectedUsersStatistics = function _getConnectedUsersStatistics() {
+	const statistics = {};
+	const totalUsers = Meteor.users.find().count();
+
+	statistics.onlineUsers = Meteor.users.find({ statusConnection: 'online' }).count();
```

Comment:
Why keep the find above, inside the get, if they will be overridden later?

Reply:
Fixed.